components/search/MoreFiltersDropdown.vue

```diff
@@ -148,8 +153,18 @@
       },
 
       dateFilterSelected(facetName, dateRange) {
```

Comment:
Change this function's logic to:
```js
        let dateQuery = [];

        if (dateRange.specific) {
          if (dateRange.start) {
            dateQuery = [dateRange.start];
          }
        } else if (dateRange.start || dateRange.end) {
          dateQuery = [rangeToQueryParam(dateRange)];
        }

        this.isCheckedSpecificDate = dateRange.specific;

        this.updateSelected(facetName, dateQuery);
```
It's a bit more concise and supports the above changes to emitDateForm.